atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,292 @@
+import uuid
```

Ruff (0.11.9)

Invalid module name: 'PipelineWorkflows'

(N999)